src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -636,7 +636,7 @@ s32 func_8088DCA4(EnElf* this) {
 }
 
 void func_8088DD34(EnElf* this, PlayState* play) {
-    Player* refActor = GET_PLAYER(play);
+    Player* player = GET_PLAYER(play);
```

Comment:
```suggestion
    Actor* playerActor = &GET_PLAYER(play)->actor;
```
Looks like all instances of this temp just access actor variables. Than you could rename `player2` to just `player`